### DISCUSSION TITLE
Doubt related testing healthFactor

### DISCUSSION THREAD

**Discussion by hoBabu1** [Cyfrin/foundry-full-course-cu#1989] [Q&A] [answered]
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/128292819/18c798af-c4af-4491-aebd-edb67c5390b4)

    
    Is this the right approach to test health factor ?
    Patrick have used different approach to test it .

**Comment by EngrPips** [comment]
Get into the habit of pasting codes instead of pictures. Your code seems right, but a code is never right until you run it and do what it is supposed to do.

**Reply by hoBabu1** [reply]
> Get into the habit of pasting codes instead of pictures. Your code seems right, but a code is never right until you run it and do what it is supposed to do.

Its working . 
I have tested it .

**Reply by EngrPips** [reply, marked as answer]
Good for you! Keep crushing it.

**Comment by Aaryan-Urunkar** [comment]
This is a decent enough test. Do not doubt yourself. It's the right approach!

**Reply by EngrPips** [reply]
indeed.
